convert_h5file.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# encoding: utf8
+
+'''
+Conversion script to convert files from release
+version 0.0.1 to 1.0.
+
+Usage: convert_h5file [-h|--help] <filename> [--save-backup] [-v|--verbose]
+
+Options:
+    --save-backup   save backup version of file in old file format [default: False].
+    -v, --verbose  print informative output to screen
+    -h, --help      print this text
+'''
+
+import docopt
+import wrapper as h5w
+import urllib
+import tarfile
+import numpy as np
+import os
+
+
+def dict_check_for_numpy_types(d):
+    '''
+    Convert all numpy datatypes to default datatypes in a dictionary.
+    '''
+    for key, value in d.items():
+        if isinstance(value, dict):
+            dict_check_for_numpy_types(value)
+        elif isinstance(value, (np.int)):
+            d[key] = int(value)
+        elif isinstance(value, (np.float)):
+            d[key] = float(value)
+
+            
+def get_previous_version(version):
+    base_url = "https://github.com/INM-6/h5py_wrapper/archive/v"
+    urllib.urlretrieve(''.join((base_url, version, ".tar.gz")),
+                       filename=''.join((os.getcwd(), version, '.tar.gz')))
+
+    with tarfile.open(''.join(('v', version, '.tar.gz'))) as f:
+        f.extract(''.join(('h5py_wrapper-', version, '/wrapper.py')))
+        f.extract(''.join(('h5py_wrapper-', version, '/__init__.py')))
+    os.rename('-'.join(('h5py_wrapper', version)),
+              '_'.join(('h5py_wrapper', version.replace('.', ''))))
+
+
+if __name__ == '__main__':
+    # First get 0.0.1 release version
```

Comment:
would it make sense to make the version a command line argument, that by defaut converts from 0.0.1 to 1.0?